pages/index.tsx

```diff
@@ -1,23 +1,83 @@
 import dynamic from 'next/dynamic';
-import Layout from 'components/Layout';
-import { Button, CircularProgress } from '@material-ui/core';
+import {
+  TextField,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  InputAdornment,
+} from '@material-ui/core';
 // import { signIn, signOut, useSession } from 'next-auth/client';
-// import styles from '../styles/Home.module.css';
+import SearchIcon from '@material-ui/icons/Search';
+import Layout from 'components/Layout';
+import styles from '../styles/Home.module.css';
 
-const MapVisual = dynamic(() => import('../components/Map'), {
+const Map = dynamic(() => import('../components/Map'), {
   ssr: false,
 });
 
 const Home: React.FC = () => {
   return (
-    <Layout title="NBJC Home">
-      <h1>Home</h1>
-      <p>FRED IS THE IMPOSTER!!!!!</p>
-      <Button variant="contained" color="primary">
-        Hello
-      </Button>
-      <CircularProgress />
-      <MapVisual />
+    <Layout>
+      <div className={styles.pageFlex}>
+        <TextField
+          id="outlined-size-small"
+          placeholder="Explore Organizations"
+          size="normal"
+          fullWidth
+          InputProps={{
+            startAdornment: (
+              <InputAdornment position="start">
+                <SearchIcon />
+              </InputAdornment>
+            ),
+          }}
+          variant="outlined"
+        />
+
+        <div className={styles.pageContent}>
+          <div className={styles.leftCol}>
+            <div className={styles.keywords}>
+              <div className={styles.keyButtonSpace}>
+                <FormControl className={styles.keyword} variant="outlined">
+                  <InputLabel>Keyword</InputLabel>
+                  <Select>
```

Comment:
need to add a `label` prop that is the same as the `InputLabel` value

Reply:
I'm not exactly sure if I did these correctly, but I just wanted to ask just in case :))!
<img width="396" alt="Screen Shot 2020-10-21 at 10 27 55 AM" src="https://user-images.githubusercontent.com/60284767/96755870-1e506f80-1388-11eb-8e9f-6061d8eb4d1d.png">
<img width="358" alt="Screen Shot 2020-10-21 at 10 28 02 AM" src="https://user-images.githubusercontent.com/60284767/96755882-20b2c980-1388-11eb-9181-a7e739f80746.png">

Reply:
I ended up pushing a commit now just in case you wanted to look at the changes immediately instead of waiting for me to push, but lmk if I still need to fix more things!

Reply:
ah it would just be `label="Keyword"`

Reply:
oh you want the `label` prop in the Select component

Reply:
the comment i add usually refers to the last line of the preview

Reply:
ohhhhh okie gotcha, thanks for letting me know!!